modules/userInterface.js

```diff
@@ -0,0 +1,42 @@
+export default class UI {
+  constructor() {
+    this.titleInput = document.getElementById('title');
+    this.authorInput = document.getElementById('author');
+    this.addBtn = document.getElementById('add');
+    this.bookList = document.getElementById('book-list');
+    this.addBtn.addEventListener('click', () => {
+      const title = this.titleInput.value;
+      const author = this.authorInput.value;
+      if (title && author) {
+        this.addBook(title, author);
+        this.titleInput.value = '';
+        this.authorInput.value = '';
+      }
+    });
+  }
+
+  displayBooks(books) {
```

Comment:
- [ ] I would implore you to make this use the arrow function just as others so that it would follow the latest industry standard.

Reply:
i connected this one by adding the arrow function